pyodide_build/common.py

```diff
@@ -30,11 +32,48 @@
 
 
 def xbuildenv_dirname() -> str:
-    from pyodide_build import __version__
+    try:
+        from pyodide_build import __version__
+    except ImportError:
+        __version__ = "0.0.0"
```

Comment:
should we also check for `ModuleNotFoundError`?
```suggestion
    try:
        from pyodide_build import __version__
    except (ImportError, ModuleNotFoundError):
        __version__ = "0.0.0"
```